.github/workflows/deploy_main.yml

```diff
@@ -79,11 +83,13 @@ jobs:
       - name: Invalidate cache on AWS CDNs
         uses: chetan/invalidate-cloudfront-action@master
         env:
-          DISTRIBUTION: "ED03LPVC4OXSW" # TODO: update to new CDN after access is provided
+          DISTRIBUTION: ${{env.WEB_CDN_ID}}
           PATHS: "/*"
           AWS_REGION: "us-east-1"
           AWS_ACCESS_KEY_ID: ${{ secrets.CLIENT_DEV_AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.CLIENT_DEV_AWS_SECRET_ACCESS_KEY }}
+      - name: Remove staging artifacts
+        run: aws s3 rm s3://usds-geoplatform-justice40-website/justice40-tool/ --exclude "*" --include "$(echo $PR_NUMBER)-*" --recursive
       - name: Update PR with deployed URL
```

Comment:
Do we want this in the `deployed_main` action? I think it should only be in the staging one since it's updating an open PR (and for deploying main there would be no open PR).

Reply:
(I realize that this isn't a change in this PR though, so we can address separately if it's too much scope creep)

Reply:
If we were to place it in the staging and if the PR was still open, the previous staging links would not load. Sometimes it's helpful to compare staging links on a given PR